portfolio/src/main/webapp/index.html

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html>
+<html lang= "en">
```

Comment:
I would suggest running the project through an HTML formatter to fix up the formatting inconsistencies.

Reply:
Done